packages/core/entity/SearchParams/index.ts

```diff
@@ -177,12 +177,29 @@ export default class FileExplorerURL {
     }
 
     /**
-     * Decode a previously encoded FileExplorerURL into components that can be rehydrated into the
+     * Decode a previously encoded SearchParams into components that can be rehydrated into the
      * application state
      */
-    public static decode(encodedURL: string): FileExplorerURLComponents {
+    public static decode(encodedURL: string): SearchParamsComponents {
         const params = new URLSearchParams(encodedURL.trim());
+        if (params.getAll("url").length > 0) {
+            return SearchParams.decodeSimpleParams(params);
+        }
+        return SearchParams.decodeComplexParams(params);
+    }
+
+    private static decodeSimpleParams(params: URLSearchParams): SearchParamsComponents {
+        const unparsedURLs = params.getAll("url");
+        return {
+            ...EMPTY_QUERY_COMPONENTS,
+            sources: unparsedURLs.map((uri) => ({
+                uri,
+                ...getNameAndTypeFromSourceUrl(uri),
+            })),
```

Comment:
If there are other params present will they be ignored? e.g., http://localhost:9001/app?url=https://biofile-finder-datasets.s3.us-west-2.amazonaws.com/Variance%20Paper%20Dataset.csv&group=Cell+Line

Reply:
Yea - for more complex stuffs they'll need to use the other system (or maybe we can add on later TBD)